src/sources/kubernetes_logs/mod.rs

```diff
@@ -991,6 +1003,10 @@ const fn default_delay_deletion_ms() -> Duration {
     Duration::from_millis(60_000)
 }
 
+const fn default_rotate_wait_sec() -> Duration {
+    Duration::from_secs(u64::MAX/2)
+}
```

Comment:
Same suggestion as above.

```suggestion
const fn default_rotate_wait() -> Duration {
    Duration::MAX
}
```